### DISCUSSION TITLE
Plex Server Support

### DISCUSSION THREAD

**Discussion by hoorrus** [PaulWoitaschek/Voice#1179] [Ideas]
I have been using the app for a while now and have loved the app. Just wondering if Plex Server Support is in your roadmap?

**Comment by rinormaloku** [comment]
What is a plex server? What would support for it look like?

**Comment by justeepin** [comment]
You'd be able to connect a Plex Library with audiobooks. You'd have an option to stream the book or download for offline access.

**Reply by hoorrus** [reply]
Thank you for your reply. Would this be an upcoming feature or something that I can configure now?

**Reply by sthompson-celerity** [reply]
I feel like what you are asking for kids to support dlna, not necessarily plex, or network folders

**Reply by hoorrus** [reply]
Actually, I am looking for something that has features similar to that of Chronicle Audiobook Player (https://play.google.com/store/apps/details?id=io.github.mattpvaughn.chronicle&hl=en_IN&gl=US). It can access files from my Plex Server, but the app is buggy. And Voice is one of the best audiobook players I have ever uses, so wondering if this will be supported.
